presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -615,7 +615,19 @@ public void close()
         ImmutableListMultimap.Builder<Integer, ColumnStatistics> encryptedStatsBuilder = ImmutableListMultimap.builder();
         for (int i = 0; i < fileStats.size(); i++) {
             if (dwrfEncryptionInfo.getGroupByNodeId(i).isPresent()) {
-                encryptedStatsBuilder.put(dwrfEncryptionInfo.getGroupByNodeId(i).get(), fileStats.get(i));
+                ColumnStatistics stats = fileStats.get(i);
```

Comment:
let's probably add something in OrcTester to exercise footer stats, and make sure, for example, there is nothing important set in the dummy stats entry

Reply:
added file stats validation to the TestDecryption tests